src/ioctls/system.rs

```diff
@@ -187,6 +279,19 @@ impl Kvm {
     /// * `max_entries_count` - Maximum number of CPUID entries. This function can return less than
     ///                         this when the hardware does not support so many CPUID entries.
     ///
+    /// # Example
+    /// ```
+    /// #[cfg(any(target_arch = "x86", target_arch = "x86_64"))] {
+    ///     use kvm_ioctls::{Kvm, MAX_KVM_CPUID_ENTRIES};
+    ///
+    ///     let kvm = Kvm::new().unwrap();
+    ///     let mut cpuid = kvm.get_supported_cpuid(MAX_KVM_CPUID_ENTRIES).unwrap();
+    ///     let cpuid_entries = cpuid.mut_entries_slice();
+    ///     assert!(cpuid_entries.len() <= MAX_KVM_CPUID_ENTRIES);
+    ///
```

Comment:
nit: remove blank line

Reply:
Will do, waiting for more comments before doing a push.